_amenities/pools/index.md

```diff
@@ -13,19 +13,17 @@ show-title: true
 
 <img src="{{site.url}}/images/swimming1.jpg" class="float-right col-sm-4 img-thumbnail" alt="Child at Klahanie pools doing a cannonball">
 
+The Klahanie Association operates two 25-meter pools with four lanes. Lakeside Pool is open year-round. Mountainview Pool operates from Memorial Weekend until Labor day. Whether you are looking for pool schedules, swim lessons, pool employment, or pool rules, you'll find detailed information below.
+
 * [Lakeside Pool Schedule]({{site.url}}/amenities/pools/lakeside-pool.html)
 * [Mountainview Pool Schedule]({{site.url}}/amenities/pools/mountainview-pool.html)
 * [Swim Lessons]({{site.url}}/amenities/pools/swim-lessons.html)
 * [Pool rules]({{site.url}}/amenities/pools/pool-rules.html)
 
+Klahanie residents can enjoy the Klahanie pools and participate in the aquatics programs. To use the pools, create a KHOA [TeamUnify](https://www.teamunify.com/SwimLessons.jsp?_tabid_=182023&team=cmkhoaa) account and then register for individual sessions, classes, and clinics. Registration for swim sessions opens at 8:00 a.m. the day before. Registraion is required for all those who wish to enter the pools. Please bring identification, and be sure to view our <a href="covid-guidelines.html">COVID Guidelines</a>.

```

Comment:
```suggestion
Klahanie residents can enjoy the Klahanie pools and participate in the aquatics programs. To use the pools, create a KHOA [TeamUnify](https://www.teamunify.com/SwimLessons.jsp?_tabid_=182023&team=cmkhoaa) account and then register for individual sessions, classes, and clinics. Registration for swim sessions opens at 8:00 a.m. the day before. Registration is required for all those who wish to enter the pools. Please bring identification, and be sure to view our [COVID Guidelines]({{site.url}}/amenities/pools/covid-guidelines.html).

```

Reply:
@jhutchings1 Thank you for fixing the hyperlink

Reply:
I didn't. You have to accept the suggestion first